android/java/org/chromium/chrome/browser/crypto_wallet/util/ImageLoader.java

```diff
@@ -51,6 +45,6 @@ public static boolean isSvg(String url) {
     }
 
     private static boolean isValidImgUrl(String url) {
-        return !TextUtils.isEmpty(url);
+        return URLUtil.isDataUrl(url) || URLUtil.isNetworkUrl(url);
```

Comment:
Blocker: Just double checked brave-core and noticed that we support only HTTPS here, so going to block to make sure we're not supporting http URLs here.

Reply:
In case `http`, we should
1. Not load the URL
2. Try replacing it with `https` before loading?

Reply:
Okay, I'm going to replace `URLUtil.isNetworkUrl(url)` with `URLUtil.isHttpsUrl(url)`. Doing so, only HTTPS will be allowed.

Reply:
Addressed in this commit 👉 [`66efe7e` (#16671)](https://github.com/brave/brave-core/pull/16671/commits/66efe7ed33fa64c3686e62c7cea2eb020cb2270f)

Reply:
Lgtm